README.md

```diff
@@ -61,7 +61,10 @@ cargo install flip-link
 cargo install probe-run --locked
 # If you want to use elf2uf2-rs instead of probe-run, instead do...
 cargo install elf2uf2-rs --locked
+# If you want to use any of the probe-rs tools (probe-rs run, cargo-embed, probe-rs-debugger)
+cargo install probe-rs --features=cli --locked
 ```
+In case you get an error ``binary `cargo-embed` already exists``, you might need to run `cargo uninstall cargo-embed` to uninstall an older version.
```

Comment:
Why are we using non-committal language here - your only options here are to not install a new version of probe-rs, to remove cargo-embed or to use `cargo install probe-rs --features=cli --force` to overwrite.

```suggestion
If you get the error ``binary `cargo-embed` already exists`` during installation of probe-rs, run `cargo uninstall cargo-embed` to uninstall your older version of cargo-embed before trying again.
```